torchbenchmark/models/ADDING_MODELS.md

```diff
@@ -37,9 +37,16 @@ A bare miminum example you can follow is https://github.com/pytorch/benchmark/tr
 The functions you specifically need to implement are 
 1. `__init__()` which is responsible for initalizing your `nn.Module`
 2. `get_module()` which is responsible for returning the initialized `nn.Module` and an example input
-3. `train()` which is a training loop, you can return a `NotImplementedError()` if your example is inference only
+3. `train()` which is a training loop, you can return a `NotImplementedError()` if your example is inference only. If your
+   training loop can be encapsulated by a `forward()`, `backward()`, and `optimizer_step()`, you need not redefine `train()`.
+   Instead, please make sure your model provides functions `forward()`, `backward()`, and `optimizer_step()` along with an
+   attribute `self.optimizer` which will be chained together for testing, see `invoke_staged_train_test()` for details. 
```

Comment:
@chuanqi129 Any thoughts on these instructions?

Reply:
Great, thanks for the update